ElrondNetwork/elrond-go
# Change Developer Fees in epoch API metric

Changed the metric that returns the developer fees accumulated in epoch. 

*Testing procedure*:
1. start a testnet with a testing scenario that includes SC calls
2. in multiple epochs, check the `<node-url>/network/economics` and check the `erd_dev_rewards` metric. The value should match the one from node logs by searching for `dev fees in epoch` for the latest epoch
  2.1. example:  API returns `erd_dev_rewards = "485681756778000000"`.  go to any log and search for the latest `| dev fees in epoch                   | (6)     |` log and you should see the same amount
3. restart the node and check the API again to ensure the values are persistent
